vcl/language_vary.vcl

```diff
@@ -0,0 +1,3 @@
+if ( req.url.path ~ "^/[a-zA-Z]{2}(?:-[a-zA-Z]{2})?/" ) {
```

Comment:
I think this should be set for pages that *do not* match those prefixes?

Reply:
Oh, yes, you're right, that's my bad.